api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/service/AbstractAccountServiceImpl.java

```diff
@@ -293,12 +293,13 @@ public void disableAssetGroup(String pluginName) {
                         "Therefore, no updates to asset groups are required.", onlineAccounts.size());
                 return;
             }
-            String updateAssetGroupStatus = assetGroupService.updateAssetGroupStatus(pluginName, false, "admin");
-            logger.info("AssetGroup {} status {}", pluginName, updateAssetGroupStatus);
+            boolean deleteAssetGroupStatus = assetGroupService.deleteAssetGroupByGroupName(pluginName);
+            logger.info("AssetGroup deletion status is {} for {}", deleteAssetGroupStatus, pluginName);
             Integer totalUsers = userPreferencesService.updateDefaultAssetGroup(pluginName);
             logger.info("Total users have been updated with the default asset group. The new count is: {}", totalUsers);
+            assetGroupService.updatePluginDataInAssetGroup(pluginName, true);
         } catch (Exception e) {
-            logger.error("Unable to make changes to required asset groups for {}", pluginName);
+            logger.error("Unable to make changes to required asset groups for {}", pluginName, e);
```

Comment:
To what required asset groups? Either say which groups or no need to specify it. And what changes? Method is about disabling - state it in the message.
"Unable to disable asset groups for {}" - is it still correct?

Reply:
ys, updated